docs/book/src/reference/version-dependencies.md

```diff
@@ -0,0 +1,15 @@
+# Versions dependency mapping
+
+| Package | Module name | Version |
+| --- | ----------- | ------- |
+| cluster-api | [sigs.k8s.io/cluster-api](https://github.com/kubernetes-sigs/cluster-apis) | v1.2.5 |
```

Comment:
It will also be good if we can mention what go dep file we are targeting here? one in the root dir and tools